RecoVertex/BeamSpotProducer/interface/BSpdfsFcn.h

```diff
@@ -18,6 +18,10 @@ ________________________________________________________________**/
 
 #include <iostream>
 #include <string>
+#include <RVersion.h>
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 32, 4)
```

Comment:
@iarspider , this isnot working for ROOT 6.32 IBs. Are you sure this change went in to root 6.32 branch too?

Reply:
I have opened https://github.com/cms-sw/cmssw/pull/46006